lib/kitchen/verifier/pester.rb

```diff
@@ -75,11 +98,38 @@ def create_sandbox
       #
       # @return [String] a command string
       def install_command
-        return if local_suite_files.empty?
-        return if config[:use_local_pester_module]
+        # the sandbox has not yet been copied to the SUT.
+        install_command_string = <<-PS1
```

Comment:
Should we rename these? These aren't really being used to install anything at the moment.

Reply:
nope, they are method overrides from the verifier parent class.